### DISCUSSION TITLE
CubeSat Sim troubleshooting

### DISCUSSION THREAD

**Discussion by trex4fun** [alanbjohnston/CubeSatSim#142] [Ideas]
I built the first part of the main board for the cubesat simulator and it doesn't seem to be transmitting.
I can ssh to the raspberri Pi and I see the green LED on the main board light up but not the blue LED. 

I run the CubeSatSim/configuration setting adding my call sign for the transmission.

Tried to see my call sign the "ground" station using the RTL SDR and the Fox Telem software in Windows 11.

Please provide me with your suggestions on the next steps. 

Thank you.

**Comment by alanbjohnston** [comment]
Hi!  

What model of Raspberry Pi are you using?

Is your RTL-SDR ground station receiving signals? Does it have an antenna?  You could test your RTL-SDR with SDR# or another application and make sure you can receive, for example, FM radio signals around 90 - 100 MHz.  I have heard that there are driver install issues for the RTL-SDR in Windows 11, but I don't have a Windows 11 computer.

Could you copy and paste your log file outputs by running these two commands in your Pi:

`CubeSatSim/log`
`CubeSatSim/log -r`

This will tell us if your LPF or BPF is detected by the software and if it is trying to transmit.

What method did you use to write your SD card - did you download the image or did you start with a standard Pi OS image (which version?) and then run the installer?

**Reply by trex4fun** [reply]
I am using the Raspberry Pi Zero 2
Attached are the text files generated running the log and log -r commands.

I do see signals with the RTL-SDR, it does have an antenna. I can't hear anything so I am not sure its working properly.
I get similar results with Windows 10, want to try it with LINUX Mint.

I did try to listen to FM stations see signals but don't hear anything in Windows 10 or 11.

Went to Help in Fox Telem and downloaded the recommended test files, loaded them and that gives me wave forms but
I couldn't hear anything with the AF Wave file. 

Also, I created the SD card downloading the image. 

Thank you so much for your help and quick reply.
 
By the way, I plan on teaching STEM classes and this project would be great for an advance class, maybe we could eventually get one launched.

[log.txt](https://github.com/alanbjohnston/CubeSatSim/files/8064427/log.txt)
[logr.txt](https://github.com/alanbjohnston/CubeSatSim/files/8064430/logr.txt)

**Comment by trex4fun** [comment]
Hi Alan,
   I was trying to buy an raspberry pi board H board, closer to what you
used in this project but everyone is out of stock.
I tried to use a raspberry pi zero 2 which is not broadcasting.

I was able to get the RTL-SDR to work on my Windows 11 computer, verified
by picking up local FM stations.

Attached are the log files you requested.

Below is an excerpt replied on the wiki:

I am using the Raspberry Pi Zero 2
Attached are the text files generated running the log and log -r commands.


Also, I created the SD card downloading the image.

Thank you !

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Mon, Feb 14, 2022 at 2:21 PM alanbjohnston ***@***.***>
wrote:

> Hi!
>
> What model of Raspberry Pi are you using?
>
> Is your RTL-SDR ground station receiving signals? Does it have an antenna?
> You could test your RTL-SDR with SDR# or another application and make sure
> you can receive, for example, FM radio signals around 90 - 100 MHz. I have
> heard that there are driver install issues for the RTL-SDR in Windows 11,
> but I don't have a Windows 11 computer.
>
> Could you copy and paste your log file outputs by running these two
> commands in your Pi:
>
> CubeSatSim/log
> CubeSatSim/log -r
>
> This will tell us if your LPF or BPF is detected by the software and if it
> is trying to transmit.
>
> What method did you use to write your SD card - did you download the image
> or did you start with a standard Pi OS image (which version?) and then run
> the installer?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alanbjohnston/CubeSatSim/discussions/142#discussioncomment-2175691>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXYE4GYPZM5QNYAYJ7AOVDLU3FI3BANCNFSM5OKNTLLQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Bridgeway Technology Inc.
BridgewayTech.org

-- Logs begin at Mon 2022-02-14 00:23:51 EST, end at Mon 2022-02-14 17:12:55 EST. --
Feb 14 00:32:35 cubesatsim systemd[1]: Started CubeSatSim service.
Feb 14 00:32:36 cubesatsim sudo[346]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/systemctl stop rpitx
Feb 14 00:32:36 cubesatsim sudo[346]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:37 cubesatsim sudo[346]: pam_unix(sudo:session): session closed for user root
Feb 14 00:32:37 cubesatsim sudo[388]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/rm /home/pi/CubeSatSim/ready /home/pi/CubeSatSim/cwready
Feb 14 00:32:37 cubesatsim sudo[388]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:37 cubesatsim cubesatsim[310]: rm: cannot remove '/home/pi/CubeSatSim/ready': No such file or directory
Feb 14 00:32:37 cubesatsim cubesatsim[310]: rm: cannot remove '/home/pi/CubeSatSim/cwready': No such file or directory
Feb 14 00:32:37 cubesatsim sudo[388]: pam_unix(sudo:session): session closed for user root
Feb 14 00:32:37 cubesatsim cubesatsim[310]: Test bus 1
Feb 14 00:32:38 cubesatsim cubesatsim[310]: I2C Bus Tested: /dev/i2c-1
Feb 14 00:32:38 cubesatsim cubesatsim[310]: Test bus 3
Feb 14 00:32:39 cubesatsim cubesatsim[310]: I2C Bus Tested: /dev/i2c-3
Feb 14 00:32:39 cubesatsim cubesatsim[310]: Finished testing
Feb 14 00:32:39 cubesatsim sudo[408]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/systemctl restart rpitx
Feb 14 00:32:39 cubesatsim sudo[408]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:39 cubesatsim sudo[408]: pam_unix(sudo:session): session closed for user root
Feb 14 00:32:40 cubesatsim sudo[416]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/rm /home/pi/CubeSatSim/camera_out.jpg.wav
Feb 14 00:32:40 cubesatsim sudo[417]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/rm /home/pi/CubeSatSim/camera_out.jpg
Feb 14 00:32:40 cubesatsim sudo[416]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:41 cubesatsim sudo[417]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:41 cubesatsim sudo[416]: pam_unix(sudo:session): session closed for user root
Feb 14 00:32:41 cubesatsim sudo[417]: pam_unix(sudo:session): session closed for user root
Feb 14 00:32:41 cubesatsim cubesatsim[310]: Mode file /home/pi/CubeSatSim/.mode contains b
Feb 14 00:32:41 cubesatsim cubesatsim[310]: Mode is BPSK
Feb 14 00:32:41 cubesatsim cubesatsim[310]: v1 Present with UHF BPF
Feb 14 00:32:41 cubesatsim cubesatsim[310]: Tx LED Off
Feb 14 00:32:41 cubesatsim cubesatsim[310]: Power LED On
Feb 14 00:32:41 cubesatsim cubesatsim[310]: INFO: I2C bus status 0: -1 1: 1 3: 3 camera: -1
Feb 14 00:32:41 cubesatsim cubesatsim[310]: Querying payload with R to reset
Feb 14 00:32:42 cubesatsim cubesatsim[310]: Querying payload with R to reset
Feb 14 00:32:45 cubesatsim cubesatsim[310]: STEM Payload not present!
Feb 14 00:32:45 cubesatsim cubesatsim[310]:  -> Is STEM Payload programed and Serial1 set to 115200 baud?
Feb 14 00:32:45 cubesatsim cubesatsim[310]: Simulated telemetry mode!
Feb 14 00:32:45 cubesatsim cubesatsim[310]: axis: -0.160000 angle: 0.158655 v: 4.937203 i: 315.556427
Feb 14 00:32:45 cubesatsim cubesatsim[310]: axis: -0.160000 angle: 1.412141 v: 4.453358 i: 278.675507
Feb 14 00:32:45 cubesatsim cubesatsim[310]: axis: -1.000000 angle: 0.785398 v: 1.488300 i: 92.642754
Feb 14 00:32:45 cubesatsim cubesatsim[310]: batt: 4.010000 speed: 1.310000 eclipse_time: 0.000000 eclipse: 0.000000 period: 219.399994 temp: 48.849998 max: 59.740002 min: 17.510000
Feb 14 00:32:45 cubesatsim cubesatsim[310]:  BPSK Mode, bufLen: 233640,  5841 bits per frame, 1200 bits per second, 4867 ms per frame 3067 ms sample period
Feb 14 00:32:45 cubesatsim cubesatsim[310]: first time - no sleep
Feb 14 00:32:45 cubesatsim cubesatsim[310]: TX Antenna Deployed!
Feb 14 00:32:45 cubesatsim cubesatsim[310]: Opening socket!
Feb 14 00:32:45 cubesatsim cubesatsim[310]: Connection Failed
Feb 14 00:32:45 cubesatsim cubesatsim[310]: Error: Connection refused
Feb 14 00:32:45 cubesatsim cubesatsim[310]: Connection Failed
Feb 14 00:32:45 cubesatsim cubesatsim[310]: Error: Connection refused
Feb 14 00:32:45 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 17
Feb 14 00:32:45 cubesatsim cubesatsim[310]: ++++ Loop time: 0.001 sec +++++
Feb 14 00:32:45 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 00:32:48 cubesatsim cubesatsim[310]: first time - no sleep
Feb 14 00:32:48 cubesatsim cubesatsim[310]: Opening socket!
Feb 14 00:32:48 cubesatsim cubesatsim[310]: Connection Failed
Feb 14 00:32:48 cubesatsim cubesatsim[310]: Error: Connection refused
Feb 14 00:32:48 cubesatsim cubesatsim[310]: Connection Failed
Feb 14 00:32:48 cubesatsim cubesatsim[310]: Error: Connection refused
Feb 14 00:32:48 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 20
Feb 14 00:32:48 cubesatsim cubesatsim[310]: ++++ Loop time: 2.909 sec +++++
Feb 14 00:32:48 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 00:32:51 cubesatsim cubesatsim[310]: first time - no sleep
Feb 14 00:32:51 cubesatsim cubesatsim[310]: Opening socket!
Feb 14 00:32:51 cubesatsim cubesatsim[310]: Connection Failed
Feb 14 00:32:51 cubesatsim cubesatsim[310]: Error: Connection refused
Feb 14 00:32:51 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 00:32:51 cubesatsim cubesatsim[310]: socket send 2 in 1 ms bytes: 460082
Feb 14 00:32:51 cubesatsim cubesatsim[310]: socket send 3 in 1 ms bytes: 460082
Feb 14 00:32:51 cubesatsim cubesatsim[310]: socket send 4 in 0 ms bytes: 460082
Feb 14 00:32:51 cubesatsim cubesatsim[310]: socket send 5 in 1 ms bytes: 460082
Feb 14 00:32:51 cubesatsim cubesatsim[310]: socket send 6 in 87 ms bytes: 460082
Feb 14 00:32:51 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 23
Feb 14 00:32:51 cubesatsim cubesatsim[310]: ++++ Loop time: 2.990 sec +++++
Feb 14 00:32:51 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 00:32:56 cubesatsim cubesatsim[310]: Sleep period: 4612
Feb 14 00:32:57 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 00:32:57 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 28
Feb 14 00:32:57 cubesatsim cubesatsim[310]: ++++ Loop time: 5.702 sec +++++
Feb 14 00:32:57 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 00:33:01 cubesatsim cubesatsim[310]: Sleep period: 3497
Feb 14 00:33:02 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 00:33:02 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 33
Feb 14 00:33:02 cubesatsim cubesatsim[310]: ++++ Loop time: 4.502 sec +++++
Feb 14 00:33:02 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 00:33:05 cubesatsim cubesatsim[310]: Sleep period: 3515
Feb 14 17:11:49 cubesatsim cubesatsim[310]: socket send 1 6308 ms bytes: 460082
Feb 14 17:11:49 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 44
Feb 14 17:11:49 cubesatsim cubesatsim[310]: ++++ Loop time: 11.107 sec +++++
Feb 14 17:11:49 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:11:49 cubesatsim cubesatsim[310]: Sleep period: 0
Feb 14 17:11:50 cubesatsim cubesatsim[310]: socket send 1 191 ms bytes: 460082
Feb 14 17:11:50 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 45
Feb 14 17:11:50 cubesatsim cubesatsim[310]: ++++ Loop time: 1.089 sec +++++
Feb 14 17:11:50 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:11:54 cubesatsim cubesatsim[310]: Sleep period: 3595
Feb 14 17:11:55 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:11:55 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 50
Feb 14 17:11:55 cubesatsim cubesatsim[310]: ++++ Loop time: 4.504 sec +++++
Feb 14 17:11:55 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:11:58 cubesatsim cubesatsim[310]: Sleep period: 3708
Feb 14 17:11:59 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:11:59 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 54
Feb 14 17:11:59 cubesatsim cubesatsim[310]: ++++ Loop time: 4.797 sec +++++
Feb 14 17:11:59 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:03 cubesatsim cubesatsim[310]: Sleep period: 3594
Feb 14 17:12:04 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:04 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 59
Feb 14 17:12:04 cubesatsim cubesatsim[310]: ++++ Loop time: 4.503 sec +++++
Feb 14 17:12:04 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:08 cubesatsim cubesatsim[310]: Sleep period: 3708
Feb 14 17:12:09 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:09 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 64
Feb 14 17:12:09 cubesatsim cubesatsim[310]: ++++ Loop time: 4.703 sec +++++
Feb 14 17:12:09 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:12 cubesatsim cubesatsim[310]: Sleep period: 3499
Feb 14 17:12:13 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:13 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 68
Feb 14 17:12:13 cubesatsim cubesatsim[310]: ++++ Loop time: 4.601 sec +++++
Feb 14 17:12:13 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:17 cubesatsim cubesatsim[310]: Sleep period: 3514
Feb 14 17:12:18 cubesatsim cubesatsim[310]: socket send 1 0 ms bytes: 460082
Feb 14 17:12:18 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 73
Feb 14 17:12:18 cubesatsim cubesatsim[310]: ++++ Loop time: 4.795 sec +++++
Feb 14 17:12:18 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:22 cubesatsim cubesatsim[310]: Sleep period: 3498
Feb 14 17:12:22 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:22 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 78
Feb 14 17:12:22 cubesatsim cubesatsim[310]: ++++ Loop time: 4.497 sec +++++
Feb 14 17:12:22 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:26 cubesatsim cubesatsim[310]: Sleep period: 3708
Feb 14 17:12:27 cubesatsim cubesatsim[310]: socket send 1 0 ms bytes: 460082
Feb 14 17:12:27 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 82
Feb 14 17:12:27 cubesatsim cubesatsim[310]: ++++ Loop time: 4.799 sec +++++
Feb 14 17:12:27 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:31 cubesatsim cubesatsim[310]: Sleep period: 3589
Feb 14 17:12:32 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:32 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 87
Feb 14 17:12:32 cubesatsim cubesatsim[310]: ++++ Loop time: 4.598 sec +++++
Feb 14 17:12:32 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:35 cubesatsim cubesatsim[310]: Sleep period: 3608
Feb 14 17:12:37 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:37 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 92
Feb 14 17:12:37 cubesatsim cubesatsim[310]: ++++ Loop time: 4.697 sec +++++
Feb 14 17:12:37 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:40 cubesatsim cubesatsim[310]: Sleep period: 3592
Feb 14 17:12:41 cubesatsim cubesatsim[310]: socket send 1 0 ms bytes: 460082
Feb 14 17:12:41 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 96
Feb 14 17:12:41 cubesatsim cubesatsim[310]: ++++ Loop time: 4.602 sec +++++
Feb 14 17:12:41 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:45 cubesatsim cubesatsim[310]: Sleep period: 3607
Feb 14 17:12:46 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:46 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 101
Feb 14 17:12:46 cubesatsim cubesatsim[310]: ++++ Loop time: 4.696 sec +++++
Feb 14 17:12:46 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:49 cubesatsim cubesatsim[310]: Sleep period: 3498
Feb 14 17:12:50 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:50 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 106
Feb 14 17:12:50 cubesatsim cubesatsim[310]: ++++ Loop time: 4.602 sec +++++
Feb 14 17:12:50 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA
Feb 14 17:12:54 cubesatsim cubesatsim[310]: Sleep period: 3516
Feb 14 17:12:55 cubesatsim cubesatsim[310]: socket send 1 1 ms bytes: 460082
Feb 14 17:12:55 cubesatsim cubesatsim[310]: INFO: Reset Count: 25 Uptime since Reset: 110
Feb 14 17:12:55 cubesatsim cubesatsim[310]: ++++ Loop time: 4.600 sec +++++
Feb 14 17:12:55 cubesatsim cubesatsim[310]: INFO: Battery voltage:  4.50 V  Threshold  3.50 V Current:    0.0 mA Threshold:  100.0 mA


Feb 14 00:32:35 cubesatsim systemd[1]: Started CubeSatSim rpitx service.
-- Logs begin at Mon 2022-02-14 00:23:51 EST, end at Mon 2022-02-14 17:13:34 EST. --
Feb 14 00:32:35 cubesatsim systemd[1]: Started CubeSatSim rpitx service.
Feb 14 00:32:36 cubesatsim python3[308]: Version is v1 with UHF BPF
Feb 14 00:32:36 cubesatsim python3[308]: 27
Feb 14 00:32:36 cubesatsim python3[308]: True
Feb 14 00:32:36 cubesatsim python3[308]: Mode is:
Feb 14 00:32:36 cubesatsim python3[308]: b
Feb 14 00:32:36 cubesatsim python3[308]: KB3VRK
Feb 14 00:32:37 cubesatsim systemd[1]: Stopping CubeSatSim rpitx service...
Feb 14 00:32:37 cubesatsim systemd[1]: rpitx.service: Main process exited, code=killed, status=15/TERM
Feb 14 00:32:37 cubesatsim systemd[1]: rpitx.service: Succeeded.
Feb 14 00:32:37 cubesatsim systemd[1]: Stopped CubeSatSim rpitx service.
Feb 14 00:32:39 cubesatsim systemd[1]: Started CubeSatSim rpitx service.
Feb 14 00:32:39 cubesatsim python3[410]: Version is v1 with UHF BPF
Feb 14 00:32:39 cubesatsim python3[410]: 27
Feb 14 00:32:39 cubesatsim python3[410]: True
Feb 14 00:32:39 cubesatsim python3[410]: Mode is:
Feb 14 00:32:39 cubesatsim python3[410]: b
Feb 14 00:32:41 cubesatsim sudo[424]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/home/pi/rpitx/rpitx -i- -m RF -f 434.9e3
Feb 14 00:32:41 cubesatsim sudo[424]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:44 cubesatsim sudo[424]: pam_unix(sudo:session): session closed for user root
Feb 14 00:32:49 cubesatsim python3[410]: BPSK
Feb 14 00:32:49 cubesatsim python3[410]: Turning LED on/off
Feb 14 00:32:49 cubesatsim python3[410]: csdr fir_interpolate_cc: window = HAMMING
Feb 14 00:32:49 cubesatsim python3[410]: csdr fir_interpolate_cc: taps_length = 79
Feb 14 00:32:49 cubesatsim python3[410]: csdr bandpass_fir_fft_cc: window = HAMMING
Feb 14 00:32:49 cubesatsim python3[410]: csdr bandpass_fir_fft_cc: (fft_size = 1024) = (taps_length = 401) + (input_size = 624) - 1
Feb 14 00:32:49 cubesatsim python3[410]: (overlap_length = 400) = taps_length - 1
Feb 14 00:32:49 cubesatsim sudo[510]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/nc -l 8080
Feb 14 00:32:49 cubesatsim sudo[510]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:49 cubesatsim sudo[516]:     root : TTY=unknown ; PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/home/pi/rpitx/sendiq -i /dev/stdin -s 96000 -f 434.9e6 -t float
Feb 14 00:32:49 cubesatsim sudo[516]: pam_unix(sudo:session): session opened for user root by (uid=0)
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 00:32:49 cubesatsim python3[410]: Peri Base = 3f000000 SDRAM c0000000
Feb 14 17:11:47 cubesatsim python3[410]: csdr bandpass_fir_fft_cc: filter initialized, low_cut = 0.002, high_cut = 0.06

**Comment by alanbjohnston** [comment]
Hi,

Sorry, I missed your posting.  The log files look fine.  Did you get it working yet?

**Comment by trex4fun** [comment]
No problem.
The problem was that I didn't have a good setup for receiving. I used my
handheld transceiver and was able to receive the expected signal.
It looks like it's working but I am having trouble getting some parts so I
have to wait for them to test out the battery charging.

Thank you for the follow up. Your nice design ensured that this project
would be a success.

Marlon


On Wed, Jun 1, 2022 at 12:16 PM alanbjohnston ***@***.***>
wrote:

> Hi,
>
> Sorry, I missed your posting. The log files look fine. Did you get it
> working yet?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alanbjohnston/CubeSatSim/discussions/142#discussioncomment-2865421>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXYE4G7MCQY73X7GGFTZ3UTVM6EG3ANCNFSM5OKNTLLQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Bridgeway Technology Inc.
BridgewayTech.org
